README.md

```diff
@@ -1,105 +1,152 @@
-Redis FDW for PostgreSQL 9.1+
-==============================
+Redis Foreign Data Wrapper for PostgreSQL
+==========================================
 
-This PostgreSQL extension implements a Foreign Data Wrapper (FDW) for
-the Redis key/value database: http://redis.io/
+This is a foreign data wrapper (FDW) to connect [PostgreSQL](https://www.postgresql.org/)
+to [Redis](http://redis.io/) key/value database. This FDW works with PostgreSQL 10+ and confirmed with some Redis versions near 6.0.
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/2/29/Postgresql_elephant.svg" align="center" height="100" alt="PostgreSQL"/>	+	<img src="https://avatars.githubusercontent.com/u/1529926" align="center" height="100" alt="Redis"/>
```

Comment:
Don't link to images etc. on 3rd party sites as it gives those sites the IP address of visitors to our resources.

Reply:
There are no web request redirect to this resources is we meant github hosting and md web interface. This is only image sources for GitHub external artefacts cache, please ensure and verify [this similar README](https://github.com/pgspider/sqlite_fdw). 
If camo.githubusercontent.com is not welcome or this repo is not for GitHub only, I can remove this links. What will better for you, @dpage ?

Reply:
I don't fully understand what you're asking. However, the image source specified is on wikimedia.org, which means that owners of that site will see traffic to this readme whenever it is rendered. Please include any images in the repo directly, and reference them relatively, so even if someone checks out the repo and then reads the README, there is no traffic from their system to Github or Wikimedia etc.

Reply:
> However, the image source specified is on wikimedia.org, which means that owners of that site will see traffic to this readme whenever it is rendered

No. You can verify this fact. This means owners of this site sometimes have a traffic of GitHub outer artefacts cache, but not in moment of README rendering and not in every moment of a new rendering.

Reply:
Github may cache the asset and update the URL to render from their cache, but not everyone uses Github to render markdown. Anyone rendering on their local machine will almost certainly expose their IP address to Wikimedia.

Further, storing assets like that outside of the repo means that you cannot view the readme as intended if you're offline.

Reply:
Ok, files will be added in the repo. Thanks, @dpage , I have understood your reason.